js/deer-config.js

```diff
@@ -6,6 +6,7 @@ export default {
     LABEL: "title", // attribute, alternate label for properties
     CONTEXT: "deer-context", // attribute, JSON-LD @context, scoped
     ATTRIBUTION: "deer-creator", // attribute, Web Annotation `creator`, scoped
+    MOTIVATION: "deer-motivation", // attribute, Web Annotation `motivation`, scoped
```

Comment:
I don't see a change for adding evidence here...

Upon viewing the file, I found it, it must have already been in there.  However `EVIDENCE` is `nv-evidence`.  I expected `deer-evidence`.  Is this on purpose?